samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -177,39 +177,38 @@ static const char dump_config_message[] =
     "Optional. Path to JSON file to dump IE parameters, which were set by application.";
 
 static const char shape_message[] =
-    "Optional. Set shape for network input. For example, \"input1[1,3,224,224],input2[1,4]\" or \"[1,3,224,224]\""
+    "Optional. Set shape for model input. For example, \"input1[1,3,224,224],input2[1,4]\" or \"[1,3,224,224]\""
     " in case of one input size. This parameter affect model input shape and can be dynamic."
     " For dynamic dimensions use symbol `?` or '-1'. Ex. [?,3,?,?]."
     " For bounded dimensions specify range 'min..max'. Ex. [1..10,3,?,?].";
 
 static const char data_shape_message[] =
-    "Required for networks with dynamic shapes. Set shape for input blobs."
+    "Required for models with dynamic shapes. Set shape for input blobs."
     " In case of one input size: \"[1,3,224,224]\" or \"input1[1,3,224,224],input2[1,4]\"."
     " In case of several input sizes provide the same number for each input (except cases with single shape for any "
     "input):"
     " \"[1,3,128,128][3,3,128,128][1,3,320,320]\", \"input1[1,1,128,128][1,1,256,256],input2[80,1]\""
     " or \"input1[1,192][1,384],input2[1,192][1,384],input3[1,192][1,384],input4[1,192][1,384]\"."
-    " If network shapes are all static specifying the option will cause an exception.";
+    " If model shapes are all static specifying the option will cause an exception.";
 
 static const char layout_message[] =
-    "Optional. Prompts how network layouts should be treated by application. "
+    "Optional. Prompts how model layouts should be treated by application. "
     "For example, \"input1[NCHW],input2[NC]\" or \"[NCHW]\" in case of one input size.";
 
 // @brief message for enabling caching
 static const char cache_dir_message[] = "Optional. Enables caching of loaded models to specified directory. "
                                         "List of devices which support caching is shown at the end of this message.";
 
 // @brief message for single load network
-static const char load_from_file_message[] = "Optional. Loads model from file directly without ReadNetwork."
-                                             " All CNNNetwork options (like re-shape) will be ignored";
+static const char load_from_file_message[] = "Optional. Loads model from file directly without ReadModel."
+                                             " All CNNModel options (like re-shape) will be ignored";
```

Comment:
What is it? `CNNModel`???

Reply:
Artifact from my refactoring, thank you for pointing it out!